### PR TITLE
Add functionality to pass client operations from test script.

### PR DESCRIPTION
**Pull request by TheJMaster**

### CODE REVIEW

File: src/Client.cpp [hunk 1]
```diff
@@ -1,21 +1,64 @@
 #include <iostream>
 #include <boost/array.hpp>
 #include <boost/asio.hpp>
+#include <tuple>
+#include <fstream>
+#include <stdio.h>
```
Comment (LukasJoswiak) [inline src/Client.cpp]: Why is this needed?

File: src/Client.cpp [hunk 2]
```diff
@@ -1,21 +1,64 @@
...
+#include <string.h>
```
Comment (LukasJoswiak) [inline src/Client.cpp]: Why string.h and not string? What's the difference?

File: src/Client.cpp [hunk 3]
```diff
@@ -1,21 +1,64 @@
...
+   
+    std::ifstream configFile("src/Config.txt");
+    if (!configFile) {
+        std::cout << "cannot open config file" << std::endl;
+        return 1;
+    }
+   
+    char buf[255];
+    while (configFile) {
+        configFile.getline(buf, 255);
```
Comment (LukasJoswiak) [inline src/Client.cpp]: Magic number.

File: src/Client.cpp [hunk 4]
```diff
@@ -1,21 +1,64 @@
...
-      return 1;
+    if (argc != 2) {
+        std::cerr << "Usage: client <operations>" << std::endl;
+        return 1;
+    }
+    
+    // Read list of host names and ports from config file.
+    std::vector<std::tuple<const char*, const char*>> serverAddresses;
+   
+    std::ifstream configFile("src/Config.txt");
```
Comment (LukasJoswiak) [inline src/Client.cpp]: Should probably make this path a constant.

File: src/Client.cpp [hunk 5]
```diff
@@ -1,21 +1,64 @@
...
 
 using boost::asio::ip::tcp;
 
 int main(int argc, char* argv[]) {
   try {
-    if (argc != 3) {
-      std::cerr << "Usage: client <host> <port>" << std::endl;
-      return 1;
+    if (argc != 2) {
+        std::cerr << "Usage: client <operations>" << std::endl;
```
Comment (LukasJoswiak) [inline src/Client.cpp]: Use two spaces instead of four.

**Review by LukasJoswiak — CHANGES_REQUESTED** [review]

File: src/Client.cpp (hunk 1 above)
Comment (TheJMaster) [reply src/Client.cpp]: tuple was not needed, string and fstream are.

**Review by LukasJoswiak — APPROVED** [review]
